test/functional/lua/treesitter_spec.lua

```diff
@@ -711,6 +711,17 @@ static int nlua_schedule(lua_State *const lstate)
     ]]
 
     eq(range, { { 0, 0, 18, 1 } })
+
+    exec_lua [[
+        parser:set_included_ranges { { 0, 0, 17, 1 } }
+        parser:parse()
+    ]]
+
+    local range_tbl = exec_lua [[
```

Comment:
is the separate `exec_lua` request significant? because otherwise you could merge it with the previous.

Reply:
I merged them then